autopts/pybtp/btp/cap.py

```diff
@@ -17,12 +17,21 @@
 import binascii
 import logging
 import struct
+from enum import IntEnum
 
-from autopts.pybtp import defs
+from autopts.pybtp import defs, btp
+from autopts.ptsprojects.stack import get_stack
 from autopts.pybtp.btp.btp import CONTROLLER_INDEX, get_iut_method as get_iut,\
     btp_hdr_check, pts_addr_get, pts_addr_type_get
 from autopts.pybtp.btp.gap import __gap_current_settings_update
-from autopts.pybtp.types import addr2btp_ba, BTPError
+from autopts.pybtp.types import addr2btp_ba, BTPError, AdType
+
+class Uuid(IntEnum):
+    ASCS = 0x184E
+    BASS = 0x184F
+    PACS = 0x1850
+    BAAS = 0x1852
+    CAS  = 0x1853
```

Comment:
This is OK for now since it lived in another CAP file before, but ideally these would be moved to https://github.com/auto-pts/auto-pts/blob/master/autopts/pybtp/types.py but I see that file already has a string version of these... 

So we should probably defined all the UUIDs in a way that we can get both a string and integer value, but that's a separate topic